### PR TITLE
Issue no. #4730

### PR DESCRIPTION
**Pull request by gopi-trip**
## PR Description 📜
Completed the RPG Game
<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @gopi-trip ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Please make following changes 
add your game entry into main readme.md file of the repo 
add your game image in the asset folder of the gamezone

**Comment by gopi-trip** [comment]
@ishita-43 I am not sure if I added the entry right. Please advice me if I did not.

**Comment by gopi-trip** [comment]
@ishita-43 Please review and merge this

File: README.md [hunk 1]
```diff
@@ -516,6 +516,11 @@ This repository also provides one such platforms where contributers come over an
 |[Samurai_Fighting_Game](https://github.com/kunjgit/GameZone/tree/main/Games/Samurai_Fighting_Game)|
 |[Tic-tac-toe](https://github.com/kunjgit/GameZone/tree/main/Games/Tic-tac-toe)|
 |[Quest_For_Riches](https://github.com/kunjgit/GameZone/tree/main/Games/Quest_For_Riches)|
+
+| [IKnowYou-Mind-Reading-Game](https://github.com/kunjgit/GameZone/tree/main/Games/IKnowYou-Mind-Reading-Game) |
+| [Tic_Tac_Toe_Neon](https://github.com/kunjgit/GameZone/tree/main/Games/Tic_Tac_Toe_Neon) |
+| [Dragon_Slayer](https://github.com/gopi-trip/GameZone/tree/main/Games/Dragon_Slayer) |
```
Comment (ishita-43) [inline README.md:522]: Change the name to kunjgit as other did instead of your repo name to make it work

File: README.md [hunk 2]
```diff
@@ -516,6 +516,11 @@ This repository also provides one such platforms where contributers come over an
...
+=======
```
Comment (ishita-43) [inline README.md:523]: Remove this line

**Comment by gopi-trip** [comment]
Done with the change

**Comment by ishita-43** [comment]
> @ishita-43, Done with the change

==== are showing various places that need to be removed also

**Review by ishita-43 — APPROVED** [review]

**Comment by ishita-43** [comment]
Also resolve arising merge conflicts

**Comment by kunjgit** [comment]
Thank you  @gopi-trip , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**